WalletWasabi/WalletWasabi.csproj

```diff
@@ -32,6 +32,7 @@
 		<PackageReference Include="Microsoft.Win32.SystemEvents" Version="6.0.0" />
 		<PackageReference Include="NBitcoin" Version="7.0.6" />
 		<PackageReference Include="NBitcoin.Secp256k1" Version="3.0.1" />
+		<PackageReference Include="System.Reactive" Version="5.0.0" />
```

Comment:
Anyhow, removing System.Reactive, will completely make this PR useless :)

Reply:
I see. It is because the whole thing was created as a shared concept when in reality it is an strictly client side feature, in fact it is not even client-side only but a feature for one and only one kind of client (the one that we have ofc). So, move this feature to a more appropriated project and you will have reactive available there.